service/src/test/java/tds/session/services/impl/ExternalSessionConfigurationServiceIntegrationTests.java

```diff
@@ -0,0 +1,37 @@
+package tds.session.services.impl;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.util.Optional;
+
+import tds.session.ExternalSessionConfiguration;
+import tds.session.services.ExternalSessionConfigurationService;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest
+public class ExternalSessionConfigurationServiceIntegrationTests {
```

Comment:
I didn't delete the repository integration test but I didn't update it either.  Going with the way I suggested in Assessments.